code/src/index.css

```diff
@@ -1,13 +1,74 @@
 body {
+  background-color: #212529;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'Press Start 2P';
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  color: white;
+  display: flex;
+  justify-content: center;
+}
+
+main {
+  max-width: 600px;
+}
+
+.user-name {
+  word-break: break-all;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+h1 {
+  font-size: 24px;
+}
+
+main {
+  margin: 20px;
+  margin-top: 100px;
+}
+
+.nes-btn {
+  margin-top: 24px;
+}
+
+.nes-bulbasaur {
+  margin: 24px;
+}
+
+.pokemon-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.nes-field {
+  margin-bottom: 20px;
+}
+
+.name-field {
+  font-size: 14px;
+}
+
+.navigation-container {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  justify-content: space-between;
+}
+
+.coordinates {
+  font-size: 14px;
+  text-align: center;
+}
+
+
+@media (min-width: 767px) {
+  .nes-container.is-rounded.is-dark {
+    padding: 50px;    
+  }
+
+}
```

Comment:
Really nice work with your styling! Its clean and neat 😎  for the next time you could challenge yourself by styling the html elements (that does not have the nes.css classes) with styled components.